util/personal_names.py

```diff
@@ -32,27 +32,54 @@ def is_corporate_name(display_name):
     return False
 
 
-def display_name_to_sort_name(display_name):
+def display_name_to_sort_name(display_name, advanced=False):
```

Comment:
A unit test can verify that the old behavior is preserved, which should eliminate any concerns about code that expects the old behavior.